rohmu/object_storage/s3.py

```diff
@@ -173,7 +175,7 @@ def __init__(
                 config=boto_config,
                 endpoint_url=custom_url,
                 region_name=region,
-                verify=is_verify_tls,
+                verify=str(cert_path) if cert_path is not None and is_verify_tls else is_verify_tls,
```

Comment:
Would be good to add tests for that, having a mistake in the verify flag would be quite catastrophic.

Also, I'd rather have an exception if `cert_path` is not `None` and  `is_verify_tls` is `False`.

Especially since the default value of `is_verify_tls` is `False`: it's too easy to write code that just specifies a `cert_path` parameter, and silently verifies nothing.